strawberryfields/gbs/subgraph.py

```diff
@@ -183,193 +178,98 @@ def random_search(
 describing the method, while the dictionary values are callable functions corresponding to the
 method."""
 
-RESIZE_DEFAULTS = {"method": "greedy-density"}
-"""dict[str, Any]: Dictionary to specify default parameters of options for resizing
-"""
-
-OPTIONS_DEFAULTS = {"heuristic": {"method": random_search}, "resize": RESIZE_DEFAULTS}
+OPTIONS_DEFAULTS = {"heuristic": {"method": random_search}}
 """dict[str, dict[str, Any]]: Options for dense subgraph identification heuristics. Composed of a
 dictionary of dictionaries with the first level specifying the option type, selected from keys
 ``"heuristic"`` and ``"resize"``, with the corresponding value a
 dictionary of options for that type.
 """
 
 
-def resize(
-        subgraphs: Iterable, graph: nx.Graph, target: int, resize_options: Optional[dict] = None
-) -> list:
-    """Resize subgraphs to a given size.
+def resize(subgraph: list, graph: nx.Graph, sizes: Optional[list] = None) -> list:
```

Comment:
It's not clear at this stage what are valid forms of `sizes`. Is it an arbitrary list of integers from 0 to node size? For example, for a 30-node graph, can we have something like `sizes = [9, 6, 2, 27, 11]`?

Reply:
If so, then for large graphs this may become cumbersome to specify if the goal is to find subgraphs within a range. Isn't it better to specify a `min_size` and a `max_size`?

Reply:
This has been updated in the following commit: https://github.com/XanaduAI/strawberryfields/pull/209/commits/4f0f0f9b9efcf5987661ba16836e629cb1efb481